src/interpreted/interpreter.py

```diff
@@ -646,6 +719,11 @@ def visit_Attribute(self, node: Attribute) -> Object:
         obj = self.visit(node.value)
         assert obj is not None
 
+        if type(obj) is Scope:
+            scoped_result = obj.get(attribute_name)
+
+            return scoped_result
```

Comment:
Instead of storing a `Scope` type here, let's create a `Module` type that inherits from `Object`, and store the scope items in `module.attributes`.

That way we won't need to add this special case. The code below will work with module objects automatically.